### PR TITLE
[Merged by Bors] - feat: The convolution of a locally integrable function f with a sequence of bump functions converges ae to f

### PR DESCRIPTION
**Pull request by sgouezel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/Average.lean [hunk 1]
```diff
@@ -197,17 +197,19 @@ theorem laverage_mem_openSegment_compl_self [IsFiniteMeasure μ] (hs : NullMeasu
 #align measure_theory.laverage_mem_open_segment_compl_self MeasureTheory.laverage_mem_openSegment_compl_self
 
 @[simp]
-theorem laverage_const (μ : Measure α) [IsFiniteMeasure μ] [h : NeZero μ] (c : ℝ≥0∞) :
```
Comment (ocfnash) [inline Mathlib/MeasureTheory/Integral/Average.lean:200]: Do you really intend this change? Maybe it's a result of some git-foo?

I thought we standardised to `NeZero μ` after https://github.com/leanprover-community/mathlib4/pull/6048

File: Mathlib/Analysis/Calculus/BumpFunction/Convolution.lean [hunk 2]
```diff
@@ -25,6 +28,9 @@ We also provide estimates in the case if `g x` is close to `g x₀` on this ball
   If `(φ i).rOut` tends to zero along a filter `l`,
   then `((φ i).normed μ ⋆[lsmul ℝ ℝ, μ] g) x₀` tends to `g x₀` along the same filter.
 - `ContDiffBump.convolution_tendsto_right`: generalization of the above lemma.
+- `ContDiffBump.ae_convolution_tendsto_right_of_locally_integrable`: let `g` be a locally
+  integrable function. Then the convolution of `g` with a family of bump functions converges
+  almost everywhere to `g`.
```
Comment (ocfnash) [inline Mathlib/Analysis/Calculus/BumpFunction/Convolution.lean:33]:
```suggestion
  integrable function. Then the convolution of `g` with a family of bump functions with
  support tending to `0` converges almost everywhere to `g`.
```

**Review by ocfnash — APPROVED** [review]
Thanks

bors d+

File: Mathlib/MeasureTheory/Integral/Average.lean (hunk 1 above)
Comment (sgouezel) [reply Mathlib/MeasureTheory/Integral/Average.lean]: Definitely a git mistake on my side. Thanks for noticing!

**Pull request closed without merging**